fleetctl/cat.go

```diff
@@ -17,24 +17,36 @@ package main
 import (
 	"fmt"
 
-	"github.com/codegangsta/cli"
+	"github.com/spf13/cobra"
 
 	"github.com/coreos/fleet/client"
 	"github.com/coreos/fleet/schema"
 )
 
-func NewCatCommand() cli.Command {
-	return cli.Command{
-		Name:        "cat",
-		Usage:       "Output the contents of a submitted unit",
-		ArgsUsage:   "UNIT",
-		Description: `Outputs the unit file that is currently loaded in the cluster. Useful to verify the correct version of a unit is running.`,
-		Action:      makeActionWrapper(runCatUnit),
-	}
+var cmdCat = &cobra.Command{
+	Use:   "cat UNIT",
+	Short: "Output the contents of a submitted unit",
+	Long: `Outputs the unit file that is currently loaded in the cluster. Useful to verify
+the correct version of a unit is running.`,
+	Run: runWrapper(runCatUnit),
+}
+
+func init() {
+	cmdFleet.AddCommand(cmdCat)
 }
 
-func runCatUnit(c *cli.Context, cAPI client.API) (exit int) {
-	args := c.Args()
+// GetCatCommand returns the cobra command for cat
```

Comment:
why are these helper functions necessary? not clear to me why the tests can't just reference the run functions directly